### ISSUE TITLE
Cannot test sending TRX to contract method

### ISSUE THREAD

**Issue by 1099511627776** [tronprotocol/tron-box#89]
There is no equivalent of tronWeb contract.method().send({....}) in tronmox testing suit.
So there is no way of testing payable methods.
The simplest method in solidity looks like this:
```
    function appendDividends() payable external returns (bool){
        _totalDividends = _totalDividends.add(msg.value);
    }
```
According to details that were discussed here: https://discordapp.com/channels/491685925227724801/496925563387248652/547411545785630720
there are no methods that could send TRX to a contract method. I've tried this code:
```
const contract = await tronWeb.contract().at(instance.address);
const result = await contract.appendDividends().send({
        callValue: dividendAmount,
         shouldPollResponse: false
 });
```

```
const result = await instance.appendDividends({
        callValue: dividendAmount,
        from: ownerAccount
 });
```
etc did not work. The contract balance was 0 after transfer in both cases. I've even tried to wait for at least 5secs before checking balance. But no luck

P.S. Using tron-quickstart docker image v1.2.6 + latest tronbox 

P.P.S.

tried using shouldPullResponse:true
the result is:
```

  Error: the object {
  "error": "Cannot find result in solidity node"
  "transaction": {
    "raw_data": {
      "contract": [
        {
          "parameter": {
            "type_url": "type.googleapis.com/protocol.TriggerSmartContract"
            "value": {
              "call_value": 5000000
              "contract_address": "41faf79f8767af46e3f3e63ffb39f9aa3ae544f634"
              "data": "ed55aab8"
              "owner_address": "41928c9af0651632157ef27a2cf17ca72c575a4d21"
            }
          }
          "type": "TriggerSmartContract"
        }
      ]
      "expiration": 1550588274000
      "fee_limit": 1000000000
      "ref_block_bytes": "0190"
      "ref_block_hash": "b32bd9083c623d55"
      "timestamp": 1550588215724
    }
    "signature": [
      "bf9e2826e3fff1e664e8ac7284556b98fd5e91799f2bec8b73dae19dd34f07f918bf4bf4f5c488eb9c3777bdd4d7ca421a2c8df6cadebf2e45276d719ac6717800"
    ]
    "txID": "b173b309524dc152f24361cf5fd198dd92fa9a673ac5e50144a392a84d32d60e"
  }
} was thrown, throw an Error :)
```

**Comment by 1099511627776** [comment]
After some investigation. I've found that the problem may be with https://github.com/tronprotocol/docker-tron-quickstart. I've made a downgrade form 1.2.7 to 1.2.5 and all worked

**Comment by sullof** [comment]
Sorry for being so late.
The 1.2.7 was a broken version. We released the 1.2.8 immediately later.
